scripts/mail-alarm

```diff
@@ -107,18 +108,27 @@ def get_mail_language(other_config):
 
 def get_config_file():
     try:
-        return open("/etc/mail-alarm.conf").read()
+        with open("/etc/mail-alarm.conf", "r") as file:
+            return file.read()
     except:
         return default_config
 
 
 def load_mail_language(mail_language):
+    mail_language_file = ""
     try:
         mail_language_file = os.path.join(
             mail_language_pack_path, mail_language + ".json"
         )
-        with open(mail_language_file, "r") as fileh:
-            return json.load(fileh, encoding="utf-8")
+
+        # this conditional branch won't be executed, it's solely for the purpose of ensuring pass in python2 ut.
+        if sys.version_info.major == 2:
+            with open(mail_language_file, "r") as fileh:
```

Comment:
The shebang is python3 and we have this piece of code just for python2 unittest looks strange. 

The code now targeting feature branch, which will goes to nightly,  It is not likely be reverted. 
So I perfer disable the python2 unittest for this one. 
But I have no objection for current solution anyway.